px-file-upload.html

```diff
@@ -34,31 +34,31 @@
 <dom-module id="px-file-upload">
   <template>
     <style include="px-file-upload-styles"></style>
-    <div id="dropZone" class="dropZone fieldset u-p-">
-      <div id=dndInstructions class$="{{_showHideDragAndDrop(_isAdvancedUpload, disabled)}} u-pb+"><px-icon class="icon u-mr--" icon="px-utl:upload"></px-icon>[[localize(message)]]</div>
+    <div id="file-upload-dropZone" class="file-upload-dropZone fieldset u-p-">
+      <div id=dndInstructions class$="{{_showHideDragAndDrop(_isAdvancedUpload, disabled)}} u-pb+"><px-icon class="file-upload-icon u-mr--" icon="px-utl:upload"></px-icon>[[localize(message)]]</div>
       <template is="dom-if" if="{{multiple}}">
-        <table class="table table--small table--rows hidden fileTable" id="fileTable">
-          <tr>
-            <th>{{localize('Name')}}</th>
-            <th>{{localize('Size')}}</th>
-            <th>{{localize('Type')}}</th>
-            <th></th>
-          </tr>
+        <div class="hidden file-upload-fileTable" id="fileTable">
+          <div class="file-upload-tr label">
+            <div class="file-upload-th">{{localize('Name')}}</div>
+            <div class="file-upload-th">{{localize('Size')}}</div>
+            <div class="file-upload-th">{{localize('Type')}}</div>
+            <div class="file-upload-th"></div>
```

Comment:
that's to extend the header row to include the column with the X icon at the end of each row. looks like this without it:

![screen shot 2017-08-11 at 2 28 47 pm](https://user-images.githubusercontent.com/9092541/29232619-6b69f3c4-7ea1-11e7-9832-eb6c146fc6af.png)